ajax/libs/colors/package.json

```diff
@@ -1,23 +1,36 @@
 {
   "name": "colors",
-  "filename": "colors.min.css",
-  "version": "1.0",
-  "description": "Better colors for the web. A collection of skin classes for faster prototyping and nicer looking sites.",
+  "version": "3.0.0",
+  "filename": "js/colors.min.js",
   "homepage": "http://clrs.cc",
+  "description": "Better default colors for the web. A collection of skin classes for faster prototyping and nicer looking sites.",
   "keywords": [
-    "css",
     "colors",
-    "html",
-    "new"
+    "design",
+    "a11y",
+    "postcss",
+    "palette",
+    "css",
+    "oocss"
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/mrmrs/colors.git"
   },
-  "license": "MIT",
   "author": {
     "name": "@mrmrs",
     "email": "hi@mrmrs.cc",
     "url": "http://mrmrs.cc"
-  }
+  },
+  "license": "MIT",
+  "npmName": "colors.css",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/*",
+        "js/*"
```

Comment:
`css/*` and `js/*` works for current version, but I think `css/**/*` and `js/**/*` is a better choice. Because the former one can only fetch the files under `css` or `js`, can not fetch the file under sub folders. For example, there is a file path `css/images/img.jpg` then you can fetch this file using the former rule. However, it's possible that the author will add some files under `css` and `js` used in `colors.min.css` or `colors.min.js`.